x/axelarcork/keeper/genesis.go

```diff
@@ -19,9 +19,12 @@ func InitGenesis(ctx sdk.Context, k Keeper, gs types.GenesisState) {
 	}
 	k.accountKeeper.SetModuleAccount(ctx, senderAccount)
 
-	for i, config := range gs.ChainConfigurations.Configurations {
+	for _, config := range gs.ChainConfigurations.Configurations {
 		k.SetChainConfiguration(ctx, config.Id, *config)
-		k.SetCellarIDs(ctx, config.Id, *gs.CellarIds[i])
+	}
+
+	for _, cellarIDSet := range gs.CellarIds {
```

Comment:
Did we catch this in unit test?

Reply:
No, this was in part a design decision in part of the original module, but when I was creating the upgrade I wasn't sure if this pattern of trying to match the two sets using this sort of ordering would work correctly in all cases, so that prompted this change.